src/Clients/MainApp/FSpot/ThumbnailCache.cs

```diff
@@ -48,13 +46,13 @@ public class ThumbnailCache : IDisposable {
 		#region Types
 		class Thumbnail {
 			// Uri of the image source
-			public SafeUri uri;
-	
+			public SafeUri Uri;
+
 			// The uncompressed thumbnail.
-			public Pixbuf pixbuf;
+			public Pixbuf Pixbuf;
```

Comment:
Can we convert these to Properties? Syntax just looks off at first glance otherwise.